bin/BuildPackages.sh

```diff
@@ -39,7 +49,9 @@ if [ -f gapicon.bmp ]
 fi
 
 # We need any subdirectory, to test if $GAPDIR is right
-SUBDIR=`ls -d */ | head -n 1`
+# dangerous to parse output of ls
+#SUBDIR=`ls -d */ | head -n 1`
+SUBDIR=`find -L ./ -maxdepth 1 -mindepth 1 -type d | sed 's|./||' | head -n 1`
```

Comment:
I use debian stable, that comes with bash 4.3 (is ````find```` not a bash command?). For me ````find -L .```` gives the directories, but all of them are preceded by ````./````, which does not matter here, but does matter later. So I do not see how ````sed```` can be dropped. Maybe I can find some (other) bash trick to do that. 

Further, I do not understand why I should escape ````.```` in the ````sed```` invocation, where that is already in single quotes, so escaped, right? I must be missing something elementary here.